### PR TITLE
Translated program page

### PR DESCRIPTION
**Pull request by jessewoo**
## Proposed changes
Translated Program page

image

## Types of changes

- [ ] I've labeled this PR with appropriate labels (`comp/*` component, bug/feature, documentation and other less important ones).

## Checklist
_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. This is simply a reminder of what we are going to look for before merging your code._

- [ ] I've linked the proper Github Issue(s) to my PR
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests (unit, integration...etc.) that prove my fix is effective or that my change works
- [ ] I have added or updated necessary documentation (if appropriate). You can search, for instance using the [`silver searcher`](https://github.com/ggreer/the_silver_searcher) `ag <STRING>` to find all usages in the repository.
- [ ] I have notified the team of any breaking or important changes

### CODE REVIEW

File: frontend/public/locales/zh/program.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  "orientation": {
+    "title": "参加见面会",
+    "blurb": "申请后，您会收到一封有关入职时间和日期的电子邮件。如果您在申请一周之内没有收到有关入职日期的电子邮件，请[联系我们](mailto:info@internationalmentorship.org)"
+  },
+  "6weeks": {
+    "title": "6个星期",
+    "blurb": "每周一次，亲自或在线与导师会面，共6周。"
+  },
+  "decide": {
+    "title": "Y你决定",
```
Comment (douglaschan32167) [inline frontend/public/locales/zh/program.json:17]: Is this Y supposed to be here?

**Comment by jessewoo** [comment]
image

Added in 404 page.

**Comment by douglaschan32167** [comment]
Cool! Did you push that commit yet? I'm not seeing it in the changes

**Comment by jessewoo** [comment]
actually iksoo already did it

**Comment by jessewoo** [comment]
@douglaschan32167 - I used my 404 page, better description.

**Comment by douglaschan32167** [comment]
I'll merge this if we can get actual translations. I don't want to be in a position where we don't get the translations in time and then we have the site go live with google translate translations on it

**Comment by vickyadebona** [comment]
@jessewoo this branch will need to be updated with master.

**Comment by kailanjian** [comment]
@douglaschan32167 any updates on this? or anyone know the status of this? Jesse is away for this week but I can make any final changes if we need to merge this...

**Comment by douglaschan32167** [comment]
@kailanjian  We don't have the official translations yet so I'm holding off on merging this for now. We probably don't want google translate translations on the live site.

**Pull request merged**